keyboards/handwired/technicpad/info.json

```diff
@@ -0,0 +1,42 @@
+{
+    "manufacturer": "nwhirschfeld",
+    "keyboard_name": "TechnicPad",
+    "maintainer": "nwhirschfeld",
+    "bootloader": "caterina",
```

Comment:
```suggestion
    "development_board": "promicro",
```

Reply:
solved in https://github.com/qmk/qmk_firmware/pull/23238/commits/557f38498da3b8484ad4a540b8283b63eaa7fbaf